cmd/zetacore_utils/main.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	args = append(args, []string{distributionList[0].TokensDistributed.String(), "--keyring-backend", "test", "--chain-id", chainID, "--yes",
 		"--broadcast-mode", broadcastMode, "--gas=auto", "--gas-adjustment=2", "--gas-prices=0.001azeta", "--node", node}...)
 
-	cmd := exec.Command("zetacored", args...)
+	cmd := exec.Command(config.AppName, args...)
```

Comment:
Using a variable for the command name is what trigger the gosec issue.
I think we should follow the linter and continue to use the hardcoded `zetacored` for the command name